redis/commands/core.py

```diff
@@ -1855,7 +1861,7 @@ class ListCommands:
     see: https://redis.io/topics/data-types#lists
     """
 
-    def blpop(self, keys, timeout=0):
+    def blpop(self, keys: list, timeout: float) -> list:
```

Comment:
Theoretically an int works here even though the help text says **The timeout argument is interpreted as a double value specifying the maximum number of seconds to block. A timeout of zero can be used to block indefinitely.**. Can you validate - and if not change accordingly?